src/renderer/services/eth2/client/eth2ApiClient/cgEth2BeaconBlocksApi.ts

```diff
@@ -23,4 +52,28 @@ export class CgEth2BeaconBlocksApi implements ICGETH2BeaconBlocksApi {
         const blocksResponse = await this.httpClient.get<{data: Json}>(`/eth/v1/beacon/blocks/${blockId}`);
         return this.config.types.SignedBeaconBlock.fromJson(blocksResponse.data, {case: "snake"});
     };
+
+    public getBlockAttestations = async (
+        blockId: "head" | "genesis" | "finalized" | number,
+    ): Promise<BlockAttestations[] | null> => {
+        const blocksResponse = await this.httpClient.get<IGetBlockAttestations>(`/eth/v1/beacon/blocks/${blockId}`);
+        if (typeof blockId === "number" && Number(blocksResponse.data.message.slot) !== blockId) return null;
+        return blocksResponse.data.message.body.attestations.map((attestation) => ({
```

Comment:
getBlock(blockId).message.body.attestations?

Reply:
I need entire block information, if a block is missing lighthouse will return the first available block from before
then comparing a block number whit the requested block number I can conclude if is available or missing

Reply:
I think it's worth opening issue on lighthouse because this is clearly bug.

But you can still use getBlock and than compare instead of this manual json parsing and casting